js/customize-snapshots-admin.js

```diff
@@ -0,0 +1,35 @@
+/* global jQuery */
+(function( $, wp ) {
+	'use strict';
+	var component = {};
+
+	component.forkClick = function() {
+		var $forkButton = $( '#snapshot-fork' ),
+			$forkSnipper = $( '.snapshot-fork-spinner' ),
```

Comment:
We could change this to `$spinner` or at the very least fix the typo and change it to `$forkSpinner`.